index.html

```diff
@@ -831,7 +831,7 @@ <h3>INR 1700</h3>
                                 </span>
                               </div>
                               <div class="col-md-3 col-xs-12 text-center ticket-column">
-                                <a href="https://in.explara.com/e/pycon-india-2018/" target="_blank" class="button button--quidel button--round-s text center-align">Buy Now</a>
+                                <h4 class="sold-out-text center-align">Sold Out</h4>
```

Comment:
Is this class name defined in CSS already? rest looks okay.

Reply:
Yes.

Just the same as all the earlier buttons.

Link for reference: https://github.com/pythonindia/inpycon2018/blob/218a1d087a523af9dd9db59b248152f5245e0057/custom/css/buttons.css#L1518